### DISCUSSION TITLE
Shelly plug s

### DISCUSSION THREAD

**Discussion by federico89mi** [indomus/forum#1264] [Hardware & Firmware]
Ciao a tutti,
Ho acquistato la shelly plug s e vorrei interfacciarla con home assistent ma a differenza degli shelly 2.5 che sono stati visti in automatico dal mio HA, questa presa nn viene vista. Premetto che è in rete e riesco a collegarmi tramite browser all'interfaccia e ho attivato il coiot però nn me la vede. Devo forse fare qualche modifica al configuration.yaml?
Vi ringrazio a tutti

**Comment by federico89mi** [comment]
qualcuno mi sa aiutare?

**Comment by indomus** [comment]
domanda cretina: perché non lo integri via mqtt?

**Reply by riccionese** [reply]
confermerei anche io. ho diversi Shelly e tutti in  MQTT e non perdono un colpo

**Reply by federico89mi** [reply]
vi ringrazio molto, non avevo ancora configurato mqtt. Sono riuscito a far funzionare la presa!!! Avrei un'altra domanda che forse è banale però non ho idea di come risolverla.
Dopo aver creato l'entità tramite mqtt switch la vedo e la controllo però quando tento di modificarla come una qualsiasi altra entità e assegnargli una stanza non me lo fa fare. Vi allego 2 immagini che forse possono farvi capire meglio la questione (spero le vediate):
![image](https://user-images.githubusercontent.com/82476732/148810198-46aad328-df37-4c7a-907e-4c5bd1a81225.png)

![image](https://user-images.githubusercontent.com/82476732/148810386-d5e1ac9d-e70a-4618-9490-9d7a46b28386.png)

Volevo capire se c'è un modo da configurazione per impostargli a mano la stanza.

Grazie

**Reply by snakuzzo** [reply]
Leggi [QUI](https://www.home-assistant.io/integrations/switch.mqtt/#device)

**Reply by federico89mi** [reply]
Scusa :( ma non riesco a capire, ho provato così e anche senza device, ti copio parte della configurazione per farti capire:

switch:
   (-) platform: mqtt
    name: "Lampada ingresso"
    device:
      suggested_area: sala
    icon: mdi:floor-lamp

**Reply by snakuzzo** [reply]
Prima cosa... hai formattato male e devi usare il markdown per il codice (per favore correggi).
Poi...hai letto quello che ho linkato ?

![device](https://user-images.githubusercontent.com/7205371/148816838-7bb4e170-2de8-442f-953d-fe6f474a3fb9.png)

**Reply by federico89mi** [reply]
ti chiedo scusa ma sono abbastanza nuovo di questo mondo e non mi è molto chiara quella stringa sinceramente (mi spieghi come sistemare il markdown?)

**Reply by snakuzzo** [reply]
non puoi definire le aree per entità (perché quella che hai definito è un'entità...e non un dispositivo) create manualmente con platform MQTT. Le aree si possono impostare per dispositivi rilevati tramite autodiscovery

**Reply by federico89mi** [reply]
ah ok chiaro, non lo sapevo, pensavo ci fosse un altro modo. 
In realtà volevo inserire la stanza semplicemente per poter poi dire a google di spegnermi tutte le luci della sala, compresa questa entità mqtt, ho visto però che si può fare semplicemente inserendo l'oggetto tramite l'app google home quindi risolto :)
